quinn-udp/src/windows.rs

```diff
@@ -1,14 +1,28 @@
 use std::{
     io::{self, IoSliceMut},
     mem,
+    net::{IpAddr, Ipv4Addr},
     os::windows::io::AsRawSocket,
+    ptr,
     sync::Mutex,
     time::Instant,
 };
 
+use once_cell::sync::OnceCell;
 use windows_sys::Win32::Networking::WinSock;
 
-use super::{log_sendmsg_error, RecvMeta, Transmit, UdpSockRef, IO_ERROR_LOG_INTERVAL};
+use crate::EcnCodepoint;
+
+use super::{
+    cmsg::{self, CMsgHdr},
+    log_sendmsg_error, RecvMeta, Transmit, UdpSockRef, IO_ERROR_LOG_INTERVAL,
+};
+
+// Enough to store max(IP_PKTINFO + IP_ECN, IPV6_PKTINFO + IPV6_ECN) bytes (header + data) and some extra margin
+const CMSG_LEN: usize = 128;
+
+// FIXME this could use [`std::sync::OnceLock`] once the MSRV is bumped to 1.70 and upper
+static WSARECVMSG_PTR: OnceCell<WinSock::LPFN_WSARECVMSG> = OnceCell::new();
```

Comment:
Please move `const`/`static` declarations to the bottom of the module.

Reply:
Sure, do you have a doc somewhere that describe the style you follow in quinn ?

Reply:
done, can you tell me if it is fine ?

Reply:
The closest thing to a style guide we have so far is rustls: https://github.com/rustls/rustls/blob/main/CONTRIBUTING.md.

Reply:
Maybe this could be link as a new section in main README.md ?